shopper_ds/shopper.py

```diff
@@ -0,0 +1,91 @@
+"""
+This module contains device class Shopper and run method for it.
+"""
+
+# Imports
+from tango import DevState, AttrWriteType, DispLevel
+from facadedevice import Facade, proxy_attribute, proxy_command
+
+
+class Shopper(Facade):
+    """
+    This class implements Tango device server for control of shutter device (integrated shutter and stopper).
+    Each Tango device represents one shutter, which can be in open or closed.
+
+    The Tango device works on a set of four PLC attributes of type DevShort, which must be
```

Comment:
AFAIK those should be boolean attributes, not short. @wkitka can you confirm this?